ddp-workspace/projects/ddp-sdk/src/lib/services/logging.service.ts

```diff
@@ -44,9 +45,9 @@ export class LoggingService {
         return Object.keys(obj).map(key => `${key}: ${obj[key]}`).join(', ');
     }
 
-    public logToCloud(payload: string, labels?: {[key: string]: string}, severity = 'INFO'): void {
+    public logToCloud(payload: string, labels?: {[key: string]: string}, severity = 'INFO'): Observable<void> {
         if (!this.config.doCloudLogging) {
```

Comment:
I would also add check `!this.config.cloudLoggingUrl` here. 

Because when I was debugging the logic I caught a corner case when url was undefined and it was not easy to get what is wrong.

Reply:
created  [PR#956](https://github.com/broadinstitute/ddp-angular/pull/956) to fix that ^

Reply:
ef13432b0